content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -296,6 +296,36 @@ in-memory locking). Kubernetes also uses controllers to check for invalid
 assignments (e.g. due to administrator intervention) and for cleaning up allocated
 IP addresses that are no longer used by any Services.
 
+#### Some heading (figure later)
+
+When configuring a Services of type NodePort, there are two mechanisms available to avoid port collisions: subranges and the `--service-node-port-range` flag. 
+
+Subranges allow for a range of NodePort values to be allocated for a specific Service or set of Services, while the `--service-node-port-range` flag sets the range of NodePorts that can be allocated for all Services on a specific node.
+
+To use subranges, you can define a `nodePort` value within the range of the `service-node-port-range` flag when creating a Service. For example:
+
+```
+apiVersion: v1
+kind: Service
+metadata:
+  name: my-service
+spec:
+  type: NodePort
+  ports:
+  - name: http
+    port: 80
+    nodePort: 30000-32767
```

Comment:
```suggestion
    nodePort: 30000-32767
```
I'm not sure if this is how to use `ServiceNodePortStaticSubrange` but I may be wrong
I suggest a technical review

Reply:
This _isn't_ how you use a static subrange.